.github/workflows/tests.yaml

```diff
@@ -26,6 +26,31 @@ jobs:
           go fmt ./...
           git diff --exit-code
 
+      - name: Wait for rabbitmq
+        run: |
+          until curl -fs http://localhost:15672;
+            do echo "waiting for rabbit mq"
+            sleep 2
+          done
+
       - name: Build and Test
+        env:
+          POSTGRESQL_CONNECT_STRING: postgres://postgres:postgres@localhost:5432/postgres
+          RABBITMQ_CONNECT_STRING: amqp://guest:guest@localhost:5672/
+          APP_PORT: 8080
+          APP_WEBHOOK_SECRET_VALUE: fake-secret
         run: |
           go test -v -cover -race ./...
```

Comment:
I think we are doing much CI change in this PR without mentioning this in the PR description. Is it really required? 

I think we first need an alignment, because we traditionally split Unit tests and Integration tests. Now all components are unit tested and integration tested on every PR.  This does seem to be ok considering it takes only 1 minute at the moment, https://github.com/canonical/github-runner-operators/actions/runs/18271829980/job/52015543135?pr=5 , but we need to ensure this is kept fast for the future too.